lib/matplotlib/patches.py

```diff
@@ -1053,23 +1064,26 @@ def _update_path(self):
             self._path = Path(np.vstack(verts), np.hstack(codes))
 
     def get_data(self):
-        """Get `.StepPatch` values and edges."""
-        return self._values, self._edges
+        """Get `.StepPatch` values, edges and baseline."""
+        return self._values, self._edges, self._baseline
 
-    def set_data(self, values, edges=None):
+    def set_data(self, values, edges=None, baseline=None):
```

Comment:
```suggestion
    def set_data(self, values=None, edges=None, baseline=None):
```
The asymmetry would be artificial.

But we don't want to allow `set_data()`, so below
~~~
if values is None and edges is None and baseline is None:
    raise ValueError(...)
~~~